cdc/model/schema_storage.go

```diff
@@ -35,8 +35,9 @@ const (
 // TableInfo provides meta data describing a DB table.
 type TableInfo struct {
 	*model.TableInfo
-	SchemaID         int64
-	TableName        TableName
+	SchemaID  int64
+	TableName TableName
```

Comment:
I think they are in `model.TableInfo`.

Reply:
`model.TableInfo` does not contains a schema name of a table, and this `TableName` field is not introduced by this PR and is heavily used in this repo. So, may be we can refine the `TableInfo` struct in other PR if needed.